src/com/spikes2212/control/PIDLoop.java

```diff
@@ -0,0 +1,18 @@
+package com.spikes2212.control;
+
+/**
+ * An interface for PID loop controllers.
+ *
+ * @author Eran Goldstein
+ */
+public interface PIDLoop {
```

Comment:
please add a `void updateSetpoint()` and `boolean onTarge()` methods that would behave like the ones in the old `PIDController`